debian/watch

```diff
@@ -4,22 +4,34 @@ version=4
 #  https://github.com/docker/docker/tags
 
 opts=\
+component=engine,\
 dversionmangle=s/[+~](debian|dfsg|ds|deb)\d*$//,\
-uversionmangle=s/-ce//g;s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/,\
-filenamemangle=s/.+\/v(\d\S*)\.tar\.gz/docker.io_$1.orig.tar.gz/ \
-  https://uscan.tianon.xyz/github.com/docker/docker-ce \
-  https://github.com/.*/v(\d\S*)\.tar\.gz
+uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/,\
+filenamemangle=s/.+\/v(\d\S*)\.tar\.gz/docker.io_$1.orig-engine.tar.gz/ \
+  https://github.com/moby/moby/tags .*/archive/v(\d\S*).tar.gz
+
+opts=\
+component=cli,\
+uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/,\
+filenamemangle=s/.+\/v(\d\S*)\.tar\.gz/docker.io_$1.orig-cli.tar.gz/ \
+  https://github.com/docker/cli/tags .*/archive/v(\d\S*).tar.gz
+
+opts=\
+component=packaging,\
+uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/,\
+filenamemangle=s/.+\/v(\d\S*)\.tar\.gz/docker.io_$1.orig-packaging.tar.gz/ \
+  https://github.com/docker/docker-ce-packaging/tags .*/archive/v(\d\S*).tar.gz
```

Comment:
This one is missing tags, isn't it? (https://github.com/docker/docker-ce-packaging/releases)

I think we're only pulling `docker.service` and `docker.socket` from it, so maybe we should just embed those in `debian/` directly instead and update them manually?

Reply:
Right, it is missing tags, I did not notice before.

So the best thing to do would be to download `docker.service` and `docker.socker` from its `master` branch and add them into the `debian` directory? (and of course remove this component from `debian/watch`)

Reply:
I intend to squash those commits before merging this branch, I did not do it yet to make the review easier.